dev_guide/projected_volumes.adoc

```diff
@@ -145,26 +145,82 @@ spec:
       - secret:
           name: pass
 ----
++
+The `user` and `pass` values are made up and can be anything valid (base64 encoded).
```

Comment:
I don't think I'd say it quite like this. Perhaps something along the lines of, the user and pass values are for example purposes only. The examples used here are base64 encoded values "user: admin", "pass:1f2d1e2e67df".